converter/converter.go

```diff
@@ -123,5 +123,5 @@ func chooseformat(rawTemplate []byte) (format string) {
 	} else if errorJSON == nil {
 		return "YAML"
 	}
-	return "Invalid output file format. Use JSON or YAML"
+	return "Unsupported file format. The input file must be either a valid _JSON_ or _YAML_ file."
```

Comment:
Let's remove underscores from there - my fault that I put those in the first place, sorry! :wink: